internal/os_util.py

```diff
@@ -128,3 +129,8 @@ def get_file_version(filename):
     except:
         logging.exception('Error getting file version for %s', filename)
     return version
+
+def pc_name():
+    with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
```

Comment:
This should be wrapped in a try-except with a default name returned if for some reason this throws.